packages/lodestar-cli/src/options/beaconNodeOptions/api.ts

```diff
@@ -2,37 +2,37 @@ import {Options} from "yargs";
 import defaultOptions from "@chainsafe/lodestar/lib/node/options";
 
 export const apiOptions = {
-  "api.rest.api": {
-    alias: ["api.namespaces"],
+  "api.namespaces": {
+    alias: ["api.namespaces", "api.rest.api"],
     type: "array",
     choices: ["beacon", "validator"],
     defaultDescription: JSON.stringify(defaultOptions.api.rest.api),
     group: "api",
   } as Options,
 
-  "api.rest.cors": {
-    alias: ["api.cors.origin"],
+  "api.cors.origin": {
+    alias: ["api.cors.origin", "api.rest.cors"],
     type: "string",
     defaultDescription: defaultOptions.api.rest.cors,
     group: "api",
   } as Options,
 
-  "api.rest.enabled": {
-    alias: ["api.enabled"],
+  "api.enabled": {
+    alias: ["api.enabled", "api.rest.enabled"],
     type: "boolean",
     defaultDescription: String(defaultOptions.api.rest.enabled),
     group: "api",
   } as Options,
 
-  "api.rest.host": {
-    alias: ["api.host"],
+  "api.host": {
+    alias: ["api.host", "api.rest.host"],
     type: "string",
     defaultDescription: defaultOptions.api.rest.host,
     group: "api",
   } as Options,
 
-  "api.rest.port": {
-    alias: ["api.port"],
+  "api.port": {
+    alias: ["api.port", "api.rest.port"],
```

Comment:
The property names here cannot be changed, they must match the actual BeaconNodeOption types.

Reply:
so does that mean we need to restructure our altona configs going forward?  for example, right now in `testnets/altona.ts`, the structure for the `api` options are as follows:

```
  "api": {
    "enabled": true,
    "namespaces": ["beacon", "validator"],
    "host": "127.0.0.1",
    "port": 9596,
    "cors": {
      "origin": "*"
    }
  },
```

Do you mean to say that now it should look something like this instead?
```
 "api": {
    "rest": {
      "enabled": true,
      "namespaces": ["beacon", "validator"],
      "host": "127.0.0.1",
      "port": 9596,
      "cors": {
        "origin": "*"
      }
    }
  },
```

I'm not sure why we need the `.rest` part.  I thought that was just an alias for when we were still using the old `mergeBeaconOptions` method?

Reply:
To clarify, there aren't any other `api` options, so I don't see a reason to sub-categorize it with `rest`

Reply:
@3xtr4t3rr3str14l 
We've simplified the configuration to _require_ one-for-one matching of yargs `Options` to `BeaconNodeOptions`. (if there's a configuration file that doesn't match the structure of `BeaconNodeOptions` (with a few additional fields, eg: `genesisStateFile`), you can think of that as a bug)
I think what you're hoping for is a matter of renaming the options in `BeaconNodeOptions`. (in the same vein as https://github.com/ChainSafe/lodestar/issues/1226) and pushing those changes to the yargs level as well. Might add a comment there, as there are quite a few fields in `BeaconNodeOptions` that could/should be renamed.

Reply:
changes reverted

Reply:
also, made my comment in #1226 accordingly to request to remove the `api` nest